WORKSPACE

```diff
@@ -3295,8 +3295,8 @@ go_repository(
 
 go_rules_dependencies()
 
-# NOTE: Keep the version in sync with Go toolchain in GitHub action.
-go_register_toolchains(version = "1.20.10")
+# NOTE: Go version is also set in GitHub action
+go_register_toolchains(version = "1.21.7")
```

Comment:
how about bumping it also in virt-v2v workspaces? and what about github actions in order to align with it?

Reply:
> how about bumping it also in virt-v2v workspaces? 

changed those workspaces as well

> and what about github actions in order to align with it?

we can do that once we branch out Forklift 2.6